e2e/upgrade-rbd.go

```diff
@@ -21,7 +24,10 @@ var _ = Describe("RBD Upgrade Testing", func() {
 		c   clientset.Interface
 		pvc *v1.PersistentVolumeClaim
 		app *v1.Pod
+		// checkSum stores the md5sum of a file to verify uniqueness.
+		checkSum string
 	)
+	const pvcSize = "2Gi"
```

Comment:
this can fail if we change the size in clone.yaml file right?

Reply:
I don't see it failing anywhere as whatever the size might be in the PVC yamls, will get replaced by "2Gi". This size was mentioned explicitly to avoid failures due to inconsistent size in yamls. Am I missing something?